x-pack/plugins/security_solution/server/lib/telemetry/receiver.ts

```diff
@@ -445,6 +457,86 @@ export class TelemetryReceiver implements ITelemetryReceiver {
     return this.esClient.search<TelemetryEvent>(query);
   }
 
+  public async fetchDiagnosticAlertsBatch(
```

Comment:
Do we still need `fetchDiagnosticAlerts`? It seems not being used anywhere

Reply:
Thanks! Updated in [f32df7b](https://github.com/elastic/kibana/pull/172087/commits/f32df7be7ed9aa118069a125b499ed7d1d5fd4f4)